llama_index/llms/konko.py

```diff
@@ -130,18 +137,70 @@ def _get_model_name(self) -> str:
     def class_name(cls) -> str:
         return "Konko_LLM"
 
+    def _create_model_info_dict(self) -> Dict[str, ModelInfo]:
+        konko = import_konko()
+
+        models_info_dict = {}
+        if is_openai_v1():
```

Comment:
llama-index only supports openai>1.0 -- is this check needed?

Reply:
Hi Logan,

The check was implemented to ensure backward compatibility. If, in the future, llamaindex becomes compatible with OpenAI's backward compatibility standards, we can consider removing this check if necessary.